editor/components/post-publish-panel/prepublish.js

```diff
@@ -26,7 +26,7 @@ function PostPublishPanelPrepublish( {
 	return (
 		<div className="editor-post-publish-panel__prepublish">
 			<div><strong>{ hasPublishAction ? __( 'Are you ready to publish?' ) : __( 'Are you ready to submit for review?' ) }</strong></div>
-			<p>{ hasPublishAction ? __( 'Here, you can do a last-minute check up of your settings below, before you publish.' ) : __( 'When you\'re ready, submit your work for review, and an Editor will be able to approve it for you.' ) }</p>
+			<p>{ hasPublishAction ? __( 'Double-check your settings, then use the button to publish your post.' ) : __( 'When you\'re ready, submit your work for review, and an Editor will be able to approve it for you.' ) }</p>
```

Comment:
"the button" is a bit vague. Maybe `the "Publish" button`? But that's awkward. Why not:

> Double-check your settings, then click "Publish".

Short and to-the-point.

Reply:
"Click" is too desktop-centric, hence the vague wording—I'd like to stick with "use" for lack of a better term.